docs/production/upgrade-or-modify.md

```diff
@@ -266,7 +266,7 @@ instructions for other supported platforms.
    to back up the system:
 
    ```bash
-   supervisorctl stop all
+   /home/zulip/deployments/current/scripts/stop-server
```

Comment:
We should check Zulip 3.x contains this tool, if we're going to change these. But maybe be safer to just not change this file.

Reply:
Ahhh, right.  These were intentionally left because 4.x is when stop-server/start-server showed up.